src/Subpages/Resources/TheProcessPage.jsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Container, Typography } from "@material-ui/core";
+import teamBuilding from "../../Assets/team_building.svg";
+import HeroPanel from "../../Components/HeroPanel";
+import { useResourcesStyles } from "../../Styles/useResourcesStyles";
+import ListItemMobileSnap from "../../ui-kit/ListItemMobileSnap";
+import SecondaryButton from "../../ui-kit/SecondaryButton";
+import { GatherDocuments } from "./GatherDocuments";
+import { LearnMore } from "./LearnMore";
+import { ScheduleHearing } from "./ScheduleHearing";
+
+const TheProcessPage = () => {
+  const classes = useResourcesStyles();
+  return (
+    <>
+      <Container>
+        <HeroPanel
+          title={"Resources"}
+          subtitle={
+            "The process of vacating a conviction is convoluted, making it difficult for most people to navigate. That’s why we have compiled a list of resources to help you get started."
+          }
+        />
+      </Container>
+      <Box className={classes.darkBlueBackground}>
+        <GatherDocuments />
+        <Container>
+          <Typography className={classes.headingStyle} variant="h2">
+            Step 2: Determine Eligibility
```

Comment:
Steps 2 and 3 are misaligned with steps 1 and 4 (in terms of spacing/padding) 
![Screen Shot 2022-01-28 at 8 42 30 AM](https://user-images.githubusercontent.com/25576005/151587080-51f15b81-2d03-41ca-b65e-3dda3e832924.png)

Reply:
The alignment is like that in the figma. The padding/margin is should definitely be fixed though. 
![image](https://user-images.githubusercontent.com/15237338/151603865-5c50704e-2ea7-48ad-9a4a-f0a2075a43a2.png)